go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/stackitcloud/stackit-sdk-go/services/dns v0.8.4
 	github.com/stackitcloud/stackit-sdk-go/services/mongodbflex v0.11.1
 	github.com/stackitcloud/stackit-sdk-go/services/opensearch v0.10.1
-	github.com/stackitcloud/stackit-sdk-go/services/postgresflex v0.11.0
+	github.com/stackitcloud/stackit-sdk-go/services/postgresflex v0.12.0-testing-waiter
```

Comment:
Instead of using the testing version, can we make a release and use it? or is this already planned?

Reply:
I was waiting for a reply from the service team. They already did, so I'll update the sdk and then update this PR.